### DISCUSSION TITLE
Initial start point

### DISCUSSION THREAD

**Discussion by glitchyordis** [SimonBlanke/Gradient-Free-Optimizers#63] [Q&A] [answered]
Hi, can we specify the initial parameters (i.e., specify the first x and y) when running the optimization, e.g., with Hill Climbing instead of starting from random points?

**Comment by SimonBlanke** [comment, marked as answer]
Hello @glitchyordis,

yes this is possible via the "warm_start" key in the `initialize`-dictionary. Here is an example:
```python
import numpy as np
from gradient_free_optimizers import HillClimbingOptimizer

def parabola_function(para):
    loss = para["x"] * para["x"]
    return -loss

search_space = {"x": np.arange(-10, 10, 0.1)}

init_para = {
    "x": 5,
}

initialize = {"warm_start": [init_para]}

opt = HillClimbingOptimizer(search_space, initialize=initialize)
opt.search(parabola_function, n_iter=50)
```

Here is the API ref for this feature:
https://simonblanke.github.io/gradient-free-optimizers-documentation/1.5/base_optimizer/#initialize
At the moment the docs miss an example for this. I will add an example script to the docs.
